source/linux/Demo_User_Guides/TI_LVGL_Demo_User_Guide.rst

```diff
@@ -69,13 +69,43 @@ Hardware Prerequisites
 Launching the TI LVGL Demo
 **************************
 
-The demo will auto launch upon Linux booting on the EVM. Follow the below instructions to flash the SD card:
+.. ifconfig:: CONFIG_sdk in ('SITARA')
 
-1. Flash an SD card with the :file:`tisdk-default-image`. User can download the :file:`tisdk-default-image` wic image from |__SDK_DOWNLOAD_URL__|.
+   The demo will auto launch upon Linux booting on the EVM. Follow the below instructions to flash the SD card:
+
+   1. Flash an SD card with the :file:`tisdk-default-image`. User can download the :file:`tisdk-default-image` wic image from |__SDK_DOWNLOAD_URL__|.
    Please follow the instructions from here to :ref:`Flash an SD card <processor-sdk-linux-create-sd-card>`.
 
-2. Insert the flashed SD card to the board, connect the display, mouse/touch-input, ethernet cable, aux cable, jumper wire and power on the EVM.
-   The TI LVGL Demo will launch automatically when the device is fully booted.
+   .. ifconfig:: CONFIG_part_variant not in ('AM62LX')
+
+      2. Insert the flashed SD card to the board, connect the display, mouse/touch-input, ethernet cable, aux cable, and power on the EVM.
+      The TI LVGL Demo will launch automatically when the device is fully booted.
+
+   .. ifconfig:: CONFIG_part_variant in ('AM62LX')
+
+      2. Insert the flashed SD card to the board, connect the display, mouse/touch-input, ethernet cable, aux cable, jumper wire and power on the EVM.
+      The TI LVGL Demo will launch automatically when the device is fully booted.
+
+.. ifconfig:: CONFIG_sdk in ('DebianSDK')
+
+   Upon booting the EVM, Weston is launched automatically. To launch the LVGL demo, refer the following
+   instructions:
+
+   1. Flash an SD card with the :file:`tisdk-debian-trixie` wic image. User can download the wic image from |__SDK_DOWNLOAD_URL__|.
+   Please follow the instructions from here to :ref:`Flash an SD card <processor-sdk-debian-create-sd-card>`.
+
+   .. ifconfig:: CONFIG_sdk not in ('AM62LX')
+
+      2. Insert the flashed SD card to the board, connect the display, mouse/touch-input, ethernet cable, aux cable, jumper wire and power the EVM on.
+
+   .. ifconfig:: CONFIG_sdk in ('AM62LX')
+
+      2. Insert the flashed SD card to the board, connect the display, mouse/touch-input, ethernet cable, aux cable, jumper wire and power the EVM on.
+
+   3. Weston is launched on boot. Shut it down with `systemctl stop weston`.
```

Comment:
This behavior is the same on both SDKs, though I do admit that stopping it is now different.